### DISCUSSION TITLE
Multiple PAI Instances IP150

### DISCUSSION THREAD

**Discussion by ReubenTechery** [ParadoxAlarmInterface/pai#429] [Q&A] [answered]
Hi, I have a HA instance running on proxmox on an intel nuc ( I am a beginner with all of this )

I have 4 alarm panels on the same network / location and I am trying to get all 4 panels onto the one home assistant . Alarm panels are paradox / ip150. 

I have successfully got 1 panel working through MQTT & PAI but I am really struggling on getting the other 3 there. Is this possible ?

If so could someone please explain how to do that in a simple way ha. 

Thanks for your time !

**Comment by yozik04** [comment, marked as answer]
I'd run 4 separate docker containers of PAI with different mqtt base settings. I do not think it is fittable into HA OS.
